.github/workflows/upgrade-test.yaml

```diff
@@ -0,0 +1,48 @@
+name: Upgrade Test
+on:
+  pull_request:
+  push:
+    branches:
+    - main
+
+concurrency:
+  group: format('{0}-{1}', ${{ github.ref }}, 'Upgrade Test')
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Upgrade Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ~1.17
+      id: go
```

Comment:
nit: this can be removed

Reply:
it's not being highlighted, but i was mentioning line 22

Reply:
Yeah, I got that

Reply:
fixed